src/app/models/AbstractModel.ts

```diff
@@ -40,14 +45,7 @@ export abstract class AbstractModel<Model = Record<string, any>> {
    */
   public static updateAttributesKey = Symbol("update-attributes");
 
-  /**
-   * Model ID
-   */
-  public readonly id?: Id;
-
-  /**
-   * Model Identifier
-   */
+  /** Model Identifier */
```

Comment:
`ID` is an abbreviation of identifier. I'd say this doc string is misleading. Maybe ` Model type name` ? Or something like that.